src/EventStore.Core/Messages/StorageMessage.cs

```diff
@@ -37,16 +34,18 @@ public class WritePrepares : Message, IPreconditionedWriteMessage, IFlushableMes
 
 			public string EventStreamId { get; private set; }
 			public long ExpectedVersion { get; private set; }
+			public CancellationToken CancellationToken { get; }
 			public readonly Event[] Events;
 
 			public readonly DateTime LiveUntil;
```

Comment:
`LiveUntil` is no longer being used apart from `ToString`. We can choose not to affect the TCP side and only introduce the cancellation support on the gRPC side.